component/integrations/node_exporter/node_exporter.go

```diff
@@ -0,0 +1,92 @@
+package node_exporter
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/agent/component"
+	"github.com/grafana/agent/component/metrics/scrape"
+	node_integration "github.com/grafana/agent/pkg/integrations/node_exporter"
+	"github.com/prometheus/common/model"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:    "integration.node_exporter",
```

Comment:
I think the name is fine for now, but we probably want to rethink it at some point.

What is an integration? 🤷 What are the types of things which go into the integration component group?

`exporter.node` or something along those lines might make more sense down the line as we add more components.

Reply:
We have an opportunity to fix the naming with Flow. I felt like the integration terminology got a bit overused. Things like the app_agent_receiver probably deserves to be in its own component namespace separate from all the components which are Prometheus exporters. 

Generally, I'm also willing to delay naming decisions right now until we have more components, since more components will make it easier to see the emerging pattern for how we want to name things. I did want to bring this up so that we start thinking about it, though.

Reply:
I completely agree integrations are super overloaded. I'm gonna go with `exporter.node`.

Reply:
#2059 suggests specifically `prometheus.integration.node_exporter` and explains why the "exporter" terminology isn't recommended (since it collides with OpenTelemetry's usage of the term)

We can discuss what the final name should be, but I think it should be scoped in the `prometheus` namespace at least.